docs/getting-started/configure-resoto/aws.md

```diff
@@ -59,6 +59,37 @@ https://youtu.be/6_nxUM0iFx4
 3. Make your `credentials` file available to Resoto at `/home/resoto/.aws`:
 
    <Tabs groupId="install-method">
+   <TabItem value="worker-config" label="Resoto Config">
```

Comment:
Can we add a note on the Kubernetes tab, that informs the user about this option?